### PR TITLE
DEVOPS-157 DEVOPS-158 Redis and mysql ha

### PR DESCRIPTION
**Pull request by mbreitung**

### CODE REVIEW

File: scripts/get_support_bundle.sh [hunk 1]
```diff
@@ -1,29 +1,52 @@
...
+kubectl ${KUBE_OPTS} get configmap sysdigcloud-config -o yaml | grep -v password > ${LOG_DIR}/config.yaml
+
 BUNDLE_NAME=$(date +%s)_sysdig_cloud_support_bundle.tgz
 tar czf ${BUNDLE_NAME} ${LOG_DIR}
 rm -rf ${LOG_DIR}
 
 echo "Support bundle generated:" ${BUNDLE_NAME}
 
-exit 0
+exit 0
```
Comment (markyjackson-taulia) [inline scripts/get_support_bundle.sh]: Line feed

File: datastores/as_kubernetes_pods/README.md [hunk 2]
```diff
@@ -3,30 +3,194 @@
...
+
+```
+kubectl -n sysdigcloud create -f manifests/mysql/mysql-deployment.yaml
+```
+
+If testing at high scale, note the increased `max_connections` setting in
+`manifests/mysql.yaml`.
+
+This solution creates a single replica instance of MySQL. The
+implementation is very robust and based on a greatly stable MySQL release
```
[the remark this answers is not in the document]
Comment (markyjackson-taulia) [reply datastores/as_kubernetes_pods/README.md]: *word

File: README.md [hunk 3]
```diff
@@ -141,26 +148,25 @@ kubectl -n sysdigcloud create secret generic sysdigcloud-java-certs --from-file=
...
     ```
-    kubectl -n sysdigcloud create -f datastores/as_kubernetes_pods/manifests/mysql.yaml
     kubectl -n sysdigcloud create -f datastores/as_kubernetes_pods/manifests/redis/redis-deployment.yaml
+    kubectl -n sysdigcloud create -f datastores/as_kubernetes_pods/manifests/mysql/mysql-deployment.yaml
     ```
 
-4. Deploy the backend Deployment sets (worker, collector and api)
- 
+3. Wait until the above pods are in a ready state. Then deploy the backend Deployment
+ sets (worker, collector and api). Pause for 60 seconds after creating the api deployment.
```
[the remark this answers is not in the document]
Comment (mbreitung) [reply README.md]: fixed

File: README.md [hunk 4]
```diff
@@ -303,11 +315,16 @@ kubectl -n sysdigcloud delete pod <pod name>
...
-kubectl -n sysdigcloud set image deployment/sdc-collector collector=quay.io/sysdig/sysdigcloud-backend:893 
-kubectl -n sysdigcloud set image deployment/sdc-worker worker=quay.io/sysdig/sysdigcloud-backend:893 
+kubectl -n sysdigcloud set image deployment/sdc-api api=quay.io/sysdig/sysdigcloud-backend:<version>
+kubectl -n sysdigcloud set image deployment/sdc-collector collector=quay.io/sysdig/sysdigcloud-backend:<version>
+kubectl -n sysdigcloud set image deployment/sdc-worker worker=quay.io/sysdig/sysdigcloud-backend:<version>
 ```
 
+#### Upgrading/Migrating/HA Datastores 
+
+Supported migrations are in a set of dirs, one for each datastore.
```
[the remark this answers is not in the document]
Comment (mbreitung) [reply README.md]: fixed

File: datastores/as_kubernetes_pods/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (mbreitung) [reply datastores/as_kubernetes_pods/README.md]: fixed

File: datastores/as_kubernetes_pods/README.md [hunk 5]
```diff
@@ -3,30 +3,194 @@
...
+(5.6), but it doesn't have HA capabilities, meaning that if the instance
+running MySQL fails, the application will face downtime until Kubernetes
+reschedules the pod on another healthy node and re-attaches to it the
+persistent storage. This downtime can tipically last from a few seconds to
```
[the remark this answers is not in the document]
Comment (mbreitung) [reply datastores/as_kubernetes_pods/README.md]: fixed

File: datastores/as_kubernetes_pods/README.md [hunk 6]
```diff
@@ -3,30 +3,194 @@
...
+                "role": "HA",
+                "status": "ONLINE"
+            }
+        }
+    },
+    "groupInformationSourceMember": "mysql://root@sysdigcloud-mysql-cluster-0:3306"
+}
+```
+
+Other useful information can usually be gathered by looking at the
```
[the remark this answers is not in the document]
Comment (mbreitung) [reply datastores/as_kubernetes_pods/README.md]: fixed

**Pull request merged**